aws/sqs/integration/src/test/scala/com/commercetools/queue/sqs/SqsClientSuite.scala

```diff
@@ -40,9 +40,12 @@ class SqsClientSuite extends QueueClientSuite {
           .asScala
           .find(_.id == awsRegion)
           .liftTo[IO](new IllegalArgumentException(s"Cannot find any suitable AWS region from $awsRegion value!"))
-        credentials <- (string("AWS_SQS_ACCESS_KEY"), string("AWS_SQS_ACCESS_SECRET")).mapN((accessKey, accessSecret) =>
+        accessKey <- string("AWS_SQS_ACCESS_KEY")
```

Comment:
Here we're changing the way to authenticate with SQS. The only way to do it will be with oidc.